### ISSUE TITLE
`cargo-watch.exe` exited unsuccessfully

### ISSUE THREAD

**Issue by DerGroncki** [passcod/cargo-watch#38]
cargo 0.10.0-nightly (25e1301 2016-03-25)
rustc 1.9.0-nightly (d5a91e695 2016-03-26)
cargo-watch v3.1.0

OS: Windows 10 Pro

If I run "cargo watch" i get

Argument 'will' is not of the form ARG or <arg>.
error: third party subcommand `cargo-watch.exe` exited unsuccessfully

Any hints how to solve this?

Thanks.
Michael

**Comment by passcod** [comment]
It sounds like docopt changed something, or is doing something wrong on Windows. Hmm.

**Comment by DerGroncki** [comment]
Ok, I uninstalled cargo-watch and I build it from source and copied the exe-file into the project folder. Running “cargo-watch.exe” I get “Waiting for changes... Hit Ctrl-C to stop.”. From here everything is working as expected, e.g. if I change one of my source files and save it “cargo test” is called automatically.

After that I tried the other way again “cargo install cargo-watch” and it is working as well (how knows what went wrong the first time?).

Thanks.
Michael
